scripts/CEE/delete-service-account/script.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+set -o nounset
+set -o pipefail
+
+CURRENTDATE=$(date +"%Y-%m-%d %T")
```

Comment:
nit: This variable `CURRENTDATE` you've set is to log when the job starts and finishes. However, it's initialized only  at the beginning of the script. If the script takes longer time to run, the finish time may be inaccurate.

So, probably better to move the date command directly into the `start_job` and `finish_job` functions like:
```
start_job(){
    local start_date=$(date +"%Y-%m-%d %T")
    ...
}

finish_job(){
    local finish_date=$(date +"%Y-%m-%d %T")
    ...
}
```